BUILDING.md

```diff
@@ -6,32 +6,39 @@ Bazel is able to fetch most of the dependencies required to build GAPID, but cur
 
 Please see the following OS specific guides for setting up the build environment.
 
+After setting up the build environment, GAPID can be built in a terminal with:
+
+```
+cd <path-to-gapid-source>
+bazel build pkg
+```
+
+The build output will be at `<path-to-gapid-source>/bazel-bin/pkg`.
+
+---
+
 ## Windows
 
 ### Install Chocolatey
 
 [Follow these instructions](https://chocolatey.org/install) to install Chocolatey.
 
-### Install Java Runtime 8
```

Comment:
Why did you remove this?
It may not be needed for bazel, but is unfortunately needed to do anything with the Android SDK. 😦

Reply:
Done.